presto-client/src/main/java/com/facebook/presto/client/StatementClient.java

```diff
@@ -206,6 +208,11 @@ public QueryResults finalResults()
         return currentResults.get();
     }
 
+    public List<QueryError> getWarnings()
```

Comment:
Will you expose this in JDBC too? ( https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html#getWarnings() )